_episodes/01-welcome.md

```diff
@@ -109,31 +109,31 @@ become a Carpentries Maintainer. During this training, we will:
 
 ## What This Onboarding Does Not Cover
 
-The Carpentries supports over 40 official lessons across our three Lesson Programs (Data Carpentry, Library Carpentry, and Software Carpentry). 
+The Carpentries supports 48 official lessons across our three Lesson Programs (Data Carpentry, Library Carpentry, and Software Carpentry). 
```

Comment:
I agree with Vini. I think we should keep this at round numbers of "over XX" so that way it stays accurate (and we can link to the lessons page once we have that in the new website)

```suggestion
The Carpentries supports over 40 official lessons across our three Lesson Programs (Data Carpentry, Library Carpentry, and Software Carpentry). 
```